.travis.yml

```diff
@@ -14,3 +11,10 @@ rvm:
 
 before_install:
   - gem install bundler
+
+script:
+  - if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then
```

Comment:
@f2prateek For this to work, we'll have to remove the `RUN_E2E_TESTS` variable added to travis. 

Either that, or we set both `RUN_E2E_TESTS` **and** `RUNSCOPE_TOKEN` as encypted variables and remove this check.

Reply:
I can update the  RUN_E2E_TESTS to be encrypted. That seems easiest - also makes it easy to run the tests locally when needed.

Reply:
Cool, I'll remove the check then

Reply:
Added `RUN_E2E_TESTS` as an encrypted variable now.